op-node/rollup/driver/driver.go

```diff
@@ -160,6 +161,7 @@ func NewDriver(
 	drain Drain,
 	driverCfg *Config,
 	cfg *rollup.Config,
+	opCfg *params.OptimismConfig,
```

Comment:
Should we just make this config part of the rollup config?

Reply:
Made it part of the `rollup.Config` 👍🏻